maistra/scripts/update-deps.sh

```diff
@@ -89,9 +89,12 @@ function apply_local_patches() {
   sed -i 's/GO_VERSION[ ]*=.*/GO_VERSION = "host"/g' ${VENDOR_DIR}/envoy/bazel/dependency_imports.bzl
 
   pushd "${VENDOR_DIR}/com_github_luajit_luajit"
-    patch -p1 -i "${PATCHES_DIR}/luajit-s390x.patch"
-    patch -p1 -i "${PATCHES_DIR}/luajit-ppc64.patch"
-    patch -p1 -i "${PATCHES_DIR}/luajit-build-flags.patch"
+  echo "\n\n\n************* WARNING - skipping broken patches, these need to be fixed! ********** \n\n\n"
+  sleep 3
+   
+  #patch -p1 -i "${PATCHES_DIR}/luajit-s390x.patch"
+  #patch -p1 -i "${PATCHES_DIR}/luajit-ppc64.patch"
+  patch -p1 -i "${PATCHES_DIR}/luajit-build-flags.patch"
```

Comment:
This patch needs to be removed as well, as it will be part of the cherry-pick of https://github.com/maistra/envoy/pull/103.  Therefore, this entire section can be removed.

Reply:
done